container/lxd/initialisation_linux.go

```diff
@@ -79,24 +79,53 @@ func (ci *containerInitialiser) Initialise() error {
 	if err := ensureDependencies(ci.lxdSnapChannel, localSeries); err != nil {
 		return errors.Trace(err)
 	}
-	err = ci.configureLxdBridge()
-	if err != nil {
-		return errors.Trace(err)
-	}
 	proxies := proxy.DetectProxies()
 	err = ci.configureLxdProxies(proxies, ci.isRunningLocally, ci.newLocalServer)
 	if err != nil {
 		return errors.Trace(err)
 	}
 
-	output, err := ci.getExecCommand(
-		"lxd",
-		"init",
-		"--auto",
-	).CombinedOutput()
+	var output []byte
+	if ci.containerNetworkingMethod == "local" {
```

Comment:
I'm not a fan of magic string-checking at this level. The `network/containerizer/bridgepolicy.go` checks for strings, but I think we can do better.

For now, I think we can let this ride.

Reply:
We should probably make them constants, but I'm happy to do it as a separate exercise. Note that we don't really want to add more logic to the `network` package. Prefer `core/network` for suitably universal things.